reference_guide/intellij_artifacts.md

```diff
@@ -0,0 +1,107 @@
+---
+title: IntelliJ Platform Artifacts Repository
+---
+
+JetBrains maintains a public repository that hosts artifacts related to the IntelliJ Platform, such as binaries and source code. This 
+repository makes artifacts more accessible for plugin authors.
+
+The IntelliJ Platform Artifacts Repository contains:
+* [Releases repository](https://www.jetbrains.com/intellij-repository/releases/) for release and EAP versions.
+* [Snapshots repository](https://www.jetbrains.com/intellij-repository/snapshots/) for nightly successful 
+  builds of each branch.
+  
+More artifacts will be hosted in the future.
+
+[Dependencies](https://jetbrains.bintray.com/intellij-third-party-dependencies) of individual modules from the IntelliJ
+Platform are hosted at the bintray repository. These artifacts should not be used directly, but a link to this 
+repository should be added to pom.xml/build.gradle files if individual modules from the IntelliJ Platform Artifacts Repository are used.    
+
+### How to Use the IntelliJ Platform Artifacts Repository
+There are two steps to using the repository:
+1. Add the corresponding repository URL to a Maven or Gradle script.
+2. Specify the Maven coordinates for the artifact. 
+ 
+#### Add the Repository URL 
+The corresponding URL for the desired artifact needs to be added to a Maven or Gradle script:
+* For release or EAP versions, use https://www.jetbrains.com/intellij-repository/releases 
+* For snapshot or EAP snapshots, use https://www.jetbrains.com/intellij-repository/snapshots
+* For dependencies of individual modules from the IntelliJ Platform, use https://jetbrains.bintray.com/intellij-third-party-dependencies 
+
+#### Specify the Artifact
+Describing the desired artifact is done with Maven coordinates:
+* Cross-platform zip distributions of IntelliJ Platform artifacts:
+  * groupId = com.jetbrains.intellij.idea
+  * artifactId = ideaIC
+  * classifier = ""
+  * packaging = .zip
+* Sources of IntelliJ IDEA Community Edition:
+  * groupId = com.jetbrains.intellij.idea
+  * artifactId = ideaIC
+  * classifier = sources
+  * packaging = jar
+* Individual modules from the IntelliJ Platform with their dependencies. For example, to specify the jps-model-serialization module:
+  * groupId = com.jetbrains.intellij.platform
+  * artifactId = jps-model-serialization
+  * classifier = ""
+  * packaging = jar
+
+For each artifact there are several versions available. The version can be specified in one of several ways:
+* A branch build is specified as _BRANCH.BUILD[.FIX]_. For example, a branch build such as `141.233`, or a branch build with a fix such as `139.555.1`
+* Release numbers are specified as _MAJOR[.MINOR][.FIX]_. For example `14`, or `14.1`, or `14.1.1`
+* A snapshot of a branch is specified as _XXX-SNAPSHOT_. For example `142-SNAPSHOT`
+* A snapshot of an early release is specified as _BRANCH-EAP-SNAPSHOT_. For example `141-EAP-SNAPSHOT`
```

Comment:
"an early release" looks obscure. I would write something like "a snapshot of a branch from which the next EAP/release build will be produced".